playbooks/roles/insights/defaults/main.yml

```diff
@@ -215,3 +215,36 @@ insights_release_specific_debian_pkgs:
     - openjdk-7-jdk
   xenial:
     - openjdk-8-jdk
+
+INSIGHT_SWITCH_SHOW_ENGAGEMENT_FORUM_ACTIVITY: "off"
```

Comment:
Picky thing about conventions: I'd add an "S" in all these variable names so that the prefix matches the role name (`INSIGHTS_`).